<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_trem1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="text">
     <string>T1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_2">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>30</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_2">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>150</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: red}</string>
    </property>
    <property name="text">
     <string>T2</string>
    </property>
   </widget>
   <widget class="QSlider" name="Trem1Slider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 1</string>
    </property>
   </widget>
   <widget class="QSlider" name="Trem2Slider">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>340</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 2</string>
    </property>
   </widget>
   <widget class="QSlider" name="Trem3Slider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 3</string>
    </property>
   </widget>
   <widget class="QSlider" name="Trem4Slider">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>410</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>400</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 4</string>
    </property>
   </widget>
   <widget class="QSlider" name="Trem5Slider">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>450</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>440</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: orange}</string>
    </property>
    <property name="text">
     <string>T3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: brown}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#9a9996;&quot;&gt;T4&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: black}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;T5&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3_5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber0">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber1">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <zorder>label_trilho3</zorder>
   <zorder>label_trilho3_5</zorder>
   <zorder>label_trilho3_3</zorder>
   <zorder>label_trilho3_2</zorder>
   <zorder>label_trilho3_4</zorder>
   <zorder>label_trilho1</zorder>
   <zorder>label_trilho2</zorder>
   <zorder>label_trilho4</zorder>
   <zorder>label_trem1</zorder>
   <zorder>label_trilho4_2</zorder>
   <zorder>label_trilho1_2</zorder>
   <zorder>label_trilho2_2</zorder>
   <zorder>label_trem2</zorder>
   <zorder>Trem1Slider</zorder>
   <zorder>label</zorder>
   <zorder>Trem2Slider</zorder>
   <zorder>label_2</zorder>
   <zorder>Trem3Slider</zorder>
   <zorder>label_3</zorder>
   <zorder>Trem4Slider</zorder>
   <zorder>label_4</zorder>
   <zorder>Trem5Slider</zorder>
   <zorder>label_5</zorder>
   <zorder>label_trem3</zorder>
   <zorder>label_trilho2_3</zorder>
   <zorder>label_trilho2_4</zorder>
   <zorder>label_trem5</zorder>
   <zorder>label_trem4</zorder>
   <zorder>lcdNumber0</zorder>
   <zorder>lcdNumber2</zorder>
   <zorder>lcdNumber3</zorder>
   <zorder>lcdNumber4</zorder>
   <zorder>lcdNumber1</zorder>
   <zorder>lcdNumber5</zorder>
   <zorder>lcdNumber6</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>863</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
